CONTRIBUTING.md

```diff
@@ -1,37 +1,10 @@
-## Develop
+# Contributing Guidelines
 
-### Upgrade the upstream Chart.
+## Upgrading
 
-- First check the upstream changelog and take notes
-
-- Then, find a suitable version.
-
-- Add values diff changelog into `changelog` folder.
-
-- Update `Chart.yaml`:
-  - dependencies
-  - version
-  - appVersion
-
-- Run `helm dependency update`
-
-- Adjust values (If applicable)
-  - Adjust Root Values
-  - Adjust Values in [config repo](https://github.com/giantswarm/config)
-
-- Update docs with `helm-docs`
-
-- Test the Upgrade.
-
-- Release a new app version
-
-- Annonce the change to our customers.
-
-### Update documentation
-
-Chart documentation is generated with [helm-docs](https://github.com/norwoodj/helm-docs) from `values.yaml` file.
-After file modification, regenerate README.md with command:
-
-```bash
-./script/update-docs.sh
-```
+* change the `kube-prometheus-stack` upstream version in Chart dependencies (`helm/prometheus-operator-app/Chart.yaml`)
+* run `helm dependency update helm/prometheus-operator-app` to update the Chart.lock file
+* re-generate `helm/prometheus-operator-app/values.schema.json`:
+  * `helm schema-gen helm/prometheus-operator-app/values.yaml > helm/prometheus-operator-app/values.schema.json` to re-generate the file.
```

Comment:
It might be a good idea to add a link to the internal doc about this : https://intranet.giantswarm.io/docs/organizational-structure/teams/cabbage/app-updates/helm-values-schema/
Especially for people where it's not as simple as the above command

Reply:
These docs are public. If the internal doc is needed/useful for customers to understand this, it should be public as well, otherwise the first question we get will be "why can't I access this link if it is useful?"

Reply:
I added a required tooling section above instead. WDYT?

Reply:
I think we would need to create an internal folder with links for us and a contributing.md file for the public but maybe we can do that later?

Reply:
Seems like a good idea :)